mk-files/docs.mk

```diff
@@ -21,21 +21,24 @@ docs:
 	go run cmd/docs/main.go
 
 .PHONY: publish-docs
-publish-docs: docs
+publish-docs:
 	$(eval DIR=$(shell mktemp -d))
+	$(eval CLI=$(DIR)/cli)
 	$(eval CLI_RELEASE=$(DIR)/cli-release)
 	$(eval DOCS_CONFLUENT_CLI=$(DIR)/docs-confluent-cli)
 
-	git clone git@github.com:confluentinc/docs-confluent-cli.git $(DOCS_CONFLUENT_CLI) && \
-	cd $(DOCS_CONFLUENT_CLI) && \
-	git checkout -b publish-docs-v$(CLEAN_VERSION) && \
-	cd - && \
 	git clone git@github.com:confluentinc/cli-release.git $(CLI_RELEASE) && \
 	go run $(CLI_RELEASE)/cmd/releasenotes/formatter/main.go $(CLI_RELEASE)/release-notes/$(CLEAN_VERSION).json docs > $(DIR)/release-notes.rst && \
-	$(SED) -i "10r $(DIR)/release-notes.rst" $(DOCS_CONFLUENT_CLI)/release-notes.rst && \
+	git clone git@github.com:confluentinc/cli.git $(CLI) && \
+    cd $(CLI) && \
+	go run cmd/docs/main.go && \
+    cd - && \
```

Comment:
Can we change these two lines to use tab instead of spaces for consistency w/ the rest of the lines?

I would have added a suggestion, but I can't figure out how to add tabs without exiting the text window...

Reply:
oh good catch, let me fix that